cove_360/lib/threesixtygiving.py

```diff
@@ -911,7 +918,7 @@ def process(self, grant, path_prefix):
                         self.json_locations.append(id_location)
                         self.count += 1
 
-        self.heading = self.format_heading_count(self.check_text['heading'])
+        self.heading = self.format_heading_count(self.check_text['heading'], test_class_type='quality_accuracy')
         self.message = self.check_text['message'][self.grants_percentage]
 
```

Comment:
A "nice to have" would if we could start using a const for this value instead of typing it out each time.

Reply:
Really good comment, thanks! I didn't like having so much repetition but it didn't come to do it as simple as adding a constant.